### ISSUE TITLE
Please release the old version for Ventura

### ISSUE THREAD

**Issue by space-esquire** [liamrosenfeld/Iconology#7]
I am unable to download the old version of this app because I have Ventura, but the new update doesn't allow me to zoom in as much as I'd like (and how I was able to with the old version). Do you mind sending me a zip of the old version. I'd very much appreciate it. Thank you so much!

**Comment by liamrosenfeld** [comment]
Huh, I thought that the app store would handle older versions automatically. Must just be for iOS. When I have some free time, I'll try to get manual app signing set up and upload a version for download here.

**Comment by space-esquire** [comment]
Yes it does! I am on ventura so it doesn't give me the option to go to a previous version. Sorry for the inconvenience, I love this app by the way! Thanks :) Take your time.
